server/routes/index.js

```diff
@@ -990,47 +982,42 @@ router
 
 router
   .route('/api/v1/users/:uid/preferences')
-  .get(ensureUser, function (req, res) {
-    const { appDb } = req.app.locals
-    appDb
-      .collection('users')
-      .findOne(
-        { uid: req.params.uid },
-        { _id: 0, preferences: 1 },
-        function (err, data) {
-          if (err) {
-            console.log(err)
-            return res.status(502).send({})
-          } else if (!data || Object.keys(data).length === 0) {
-            return res.status(404).send({})
-          } else {
-            return res.status(200).json(data['preferences'])
-          }
-        }
-      )
+  .get(ensureUser, async (req, res) => {
+    try {
+      const { prisma } = req.app.locals
+      const userPreferences = await prisma.findUnique({
+        where: { uid: req.params.uid },
+        select: { preferences: true },
+      })
+      console.log(userPreferences)
```

Comment:
did you mean to leave this in?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/237/commits/6580f4c44fa739706b7214079ccef4af7e744830